commercial-paper/README.md

```diff
@@ -87,7 +87,15 @@ docker exec cliMagnetoCorp peer chaincode install -n papercontract -v 0 -p /opt/
 docker exec cliMagnetoCorp peer chaincode instantiate -n papercontract -v 0 -l java -c '{"Args":["org.papernet.commercialpaper:instantiate"]}' -C mychannel -P "AND ('Org1MSP.member')"
 ```
 
-> If you want to try both a Java and JavaScript Contract, then you will need to restart the infrastructure and deploy the other contract.
+**For a Go Contract:**
+
+```
+docker exec cliMagnetoCorp bash -c "cd /opt/gopath/src/github.com/hyperledger/fabric-samples/commercial-paper/organization/magnetocorp/contract-go GO111MODULE=on GOCACHE=on go mod vendor"
```

Comment:
This is missing a semi colon after the cd command, and GOCACHE=on didn't work for me - I got: build cache is required, but could not be located: GOCACHE is not an absolute path.
The following seems to work fine:
docker exec cliMagnetoCorp bash -c "cd /opt/gopath/src/github.com/hyperledger/fabric-samples/commercial-paper/organization/magnetocorp/contract-go; GO111MODULE=on go mod vendor"

Reply:
Thanks for pointing this out